src/main/resources/xml/schema/2020-1/nordic2020-1.sch

```diff
@@ -794,7 +789,7 @@
                         ()"/>
             <assert test="
                     not(matches(base-uri(.), $filename-regex)) or (for $t in tokenize(@epub:type, '\s+')

```

Comment:
This rule has changed in the new guidelines. It is now based on `role` instead of `epub:type`. See: https://format.mtm.se/nordic_epub/2020-1/#file-naming-convention

Reply:
I have replaced epub:type with role now.